src/TourPortal.js

```diff
@@ -58,6 +54,7 @@ class TourPortal extends Component {
     update: PropTypes.string,
     updateDelay: PropTypes.number,
     disableInteraction: PropTypes.bool,
+    rounded: PropTypes.number,
```

Comment:
also here with `disableNavigationDotsInteraction` and `disableKeyboardNavigation`, right?